contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.2.0--1.2.1.sql

```diff
@@ -0,0 +1,7 @@
+ -- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION ""babelfishpg_tsql"" UPDATE TO '1.2.1'" to load this file. \quit
+
+SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+-- Reset search_path to not affect any subsequent scripts
+SELECT set_config('search_path', trim(leading 'sys, ' from current_setting('search_path')), false);.
```

Comment:
The last '.' in the end needs to be removed. Otherwise upgrade will fail.

Reply:
Thanks for pointing out! Let me revise it.